solution/deps.yaml

```diff
@@ -39,7 +39,7 @@ jabba:
 pensieve-api:
   sourceRegistry: registry.scality.com/pensieve-api
   image: pensieve-api
-  tag: 1.0.0-alpha.8
+  tag: 1.0.0-rc.2
```

Comment:
just checking, is this one still tagged `rc.2`?

Reply:
Yes, it’s its own rc number independent of where it’s used. Could be declared GA soon :)

Reply:
Yes, it’s its own rc number, independent of where it’s used. Could be declared GA soon :)